### DISCUSSION TITLE
Theme?

### DISCUSSION THREAD

**Discussion by AndreasPantle** [autokey/autokey#619] [Q&A] [answered]
Ahm it is off-topic I know (Sorry) but is there someone who can tell me which theme this screenshot is from?

It is from the start page of the wiki her: [https://github.com/autokey/autokey/wiki](https://github.com/autokey/autokey/wiki)

![https://raw.githubusercontent.com/wiki/autokey/autokey/images/AutoKey_Scripting.png](https://raw.githubusercontent.com/wiki/autokey/autokey/images/AutoKey_Scripting.png)

**Comment by josephj11** [comment, marked as answer]
TL;DR: IDK, but that won't stop me from commenting. :)

What? Do you actually want to be able to see your code when you're editing it? LOL.

This is not one of AutoKey's stronger features. I always use an external editor for scripts when changing more than a few characters.

I have very minimal familiarity with the source code, so take the following with a grain of salt.

There's a  line in the source code that selects a theme. I don't remember where ATM. There was some work on exposing that in the GUI so it could be changed, but I don't know if it made it into the 0.96 beta release.

It might be this: (and I may have changed whatever the default is to "kate".)

    self.buffer.set_style_scheme(self.__sm.get_scheme("kate"))

which lives at line 618 of autokey/lib/autokey/gtkui/configwindow.py in AutoKey 0.95.10

A few lines above that is 

    self.__sm = GtkSource.StyleSchemeManager()

I didn't find corresponding code in the Qt front end which is coded quite differently.
